src/components/header/dropdown-company/dropdown-company.scss

```diff
@@ -0,0 +1,82 @@
+*{
+    text-decoration: none;
+    bottom: unset;
+    color: #fff;
+}
+.button-company{
+    padding: 10px 7px;
+    display: flex;
+    background-color: unset;
+    color: #fff;
+    border: unset;
+    font-size: 16px;
+}
+.svg-company{
+    display: flex;
+    align-items: center;
+    margin-left: 9px;
+}
+.content-dropdown{
+    display: flex;
+    align-items: center;
+    color: #000 !important;
+    font-family: "Helvetica Neue", sans-serif;
+    font-size: 18px;
+    padding: 0px 0px;
+}
+.menu-company {
+    float: right;
+    overflow: hidden;
+    width: 100%;
+    cursor: pointer;
+  
+    .show {
+      display: flex;
+      flex-direction: column;
+      border-radius: 0px 0px 12px 12px;
+      margin-top: 22px;
+    }
+  }
+  
+  .menu-company .dropdown-button {
```

Comment:
try do put the second class inside the first one, something like:

.menu-company {
    .dropdown-button {
       //your styles here
    }
}